tools/wptrunner/wptrunner/executors/executoraxapi.py

```diff
@@ -37,7 +47,11 @@ def find_active_tab(browser):
         if err:
             continue
         if role == "AXWebArea":
-            return node
+            (err, tab_url) = AXUIElementCopyAttributeValue(node, "AXURL", None)
+            if (str(tab_url) == url):
```

Comment:
```suggestion
            if err:
                continue
            if str(tab_url) == url:
```

Also, why do we need to use `str()` here?

Reply:
Very good question. It turns out that  AXUIElementCopyAttributeValue(node, "AXURL", None) returns a object, a ` <objective-c class NSURL at 0x1d...>`, so you can't compare it to a URL string, you need to convert it first! Really confused me at first, because if you print `tab_url`, it looks like a string.

Reply:
Ah, thank you for explaining! Maybe add a comment as well?

Also, did you want to explicitly handle `err`, and remove the extra parentheses?